src/test/resources/processExecution/recorder.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as child_process from "child_process";
+import * as events from "events";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+interface ISpawnOptions {
```

Comment:
If you end up not using the NodeJS built-in interface, I think you should at least reuse the `ISpawnOptions` you defined in `recording.ts` (you already import the `ISpawnArguments` interface from that file)